src/utils/release_activity_updater.rs

```diff
@@ -37,11 +37,10 @@ pub fn update_release_activity() -> Result<()> {
 
         let release_count: i64 = rows.get(0).get(0);
         let failure_count: i64 = failures_count_rows.get(0).get(0);
-        let now = now();
-        let date = now - Duration::days(day);
+        let now = Utc::now().naive_utc();
+        let date = now.checked_sub_signed(Duration::days(day)).unwrap();
```

Comment:
Why bother using `checked` if you're just going to `unwrap`?

Reply:
It's the only option for chrono types

Reply:
`DateTime` implements `Sub`: https://docs.rs/chrono/0.4.11/chrono/struct.DateTime.html#impl-Sub%3CDuration%3E

Reply:
Whoops, didn't see that